edn_format/edn_parse.py

```diff
@@ -59,7 +60,7 @@ def p_term_leaf(p):
 
 def p_vector(p):
     """vector : VECTOR_START expressions VECTOR_END"""
-    p[0] = ImmutableList(p[2])
+    p[0] = ImmutableList(list(p[2]))
```

Comment:
We may avoid this change by using `list()` in `ImmutableList`’s `__init__` function:

```diff
diff --git a/edn_format/immutable_list.py b/edn_format/immutable_list.py
index 5686847..7b934f5 100644
--- a/edn_format/immutable_list.py
+++ b/edn_format/immutable_list.py
@@ -8,7 +8,7 @@ import copy as _copy
 class ImmutableList(collections.Sequence, collections.Hashable):
     def __init__(self, wrapped_list, copy=True):
         """Returns an immutable version of the given list. Optionally creates a shallow copy."""
-        self._list = _copy.copy(wrapped_list) if copy else wrapped_list
+        self._list = list(wrapped_list) if copy else wrapped_list
         self._hash = None
 
     def __repr__(self):
```